src/shared_utilities/drivers.jl

```diff
@@ -230,17 +236,21 @@ function surface_fluxes_at_a_point(
         sc;
         tol_neutral = SFP.cp_d(surface_flux_params) / 100000,
     )
-    # Land needs a volume flux of water, not mass flux
-    vapor_flux =
-        SurfaceFluxes.evaporation(surface_flux_params, sc, conditions.Ch) /
-        _ρ_liq
-    return (
-        lhf = conditions.lhf,
-        shf = conditions.shf,
-        Ch = conditions.Ch,
-        vapor_flux = vapor_flux,
-        r_ae = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc)),
-    )
+    grav::FT = LSMP.grav(earth_param_set)
+    cp_d::FT = Thermodynamics.Parameters.cp_d(thermo_params)
+    R_d::FT = Thermodynamics.Parameters.R_d(thermo_params)
+    T_0::FT = LSMP.T_0(earth_param_set)
+    cp_m = Thermodynamics.cp_m(thermo_params, ts_in)
+    T_in = Thermodynamics.air_temperature(thermo_params, ts_in)
+    ΔT = T_in - T_sfc
+    hd_sfc = cp_d * (T_sfc - T_0) + R_d * T_0
+    E0 = SurfaceFluxes.evaporation(surface_flux_params, sc, conditions.Ch)
+    r_ae = 1 / (conditions.Ch * SurfaceFluxes.windspeed(sc))
+    E = E0 * r_ae / (r_sfc + r_ae)
+    Ẽ = E / _ρ_liq
+    H = conditions.shf + hd_sfc * (E0 - E)
```

Comment:
Could you point me to the lines where these values in `conditions` are computed?

Reply:
This correction then results in 
$ H = -\rho_{sfc} * C_{h} * |u_{h}| * (c_{pm} * \Delta T + \Delta \Phi) - (h_{d,sfc})  E  + h_{d, sfc} (E_{0} - E)$ - is this right ? 
```-ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ) - (hd_sfc) * E + hd_sfc * (E0 - E) ```

Reply:
Thanks. Could you please clarify what the rationale is for removing evaporation from sensible heat? In equation 2.5.3 here: https://escomp.github.io/ctsm-docs/versions/release-clm5.0/html/tech_note/Fluxes/CLM50_Tech_Note_Fluxes.html, sensible heat is only a function of the temperature gradient between the surface and the atmosphere and the aerodynamic resistance. I don't understand how evaporation is related to sensible heat.

Reply:
Hi Renato - I dont understand this myself, but in the atmosphere design docs, this is where it shows up:
<img width="1211" alt="Screenshot 2023-10-10 at 10 14 30" src="https://github.com/CliMA/ClimaLSM.jl/assets/65979205/e08d614f-08ef-4819-a660-7eab4953cee4">

Reply:
> `conditions`

https://github.com/CliMA/ClimaLSM.jl/blob/7fc4fb22f1cddb184ed8f08ef81c81286ce96d8c/src/shared_utilities/drivers.jl#L234

Reply:
> Could you point me to the lines where these values in `conditions` are computed? `boundary_conditions.jl` > `172` it seems.

https://github.com/CliMA/ClimaLSM.jl/blob/7fc4fb22f1cddb184ed8f08ef81c81286ce96d8c/src/shared_utilities/drivers.jl#L234

Reply:
> This correction then results in $$ H = -\rho_{sfc} * C_{h} * |u_{h}| * (c_{pm} * \Delta T + \Delta \Phi) - (h_{d,sfc}) E + h_{d, sfc} (E_{0} - E)$$ - is this right ? `-ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ) - (hd_sfc) * E + hd_sfc * (E0 - E) `

sorry @akshaysridhar ! I originally "edited" your comment instead of quoting in my reply.



not quite. E0 should be the evaporation as computed by SurfaceFluxes, without the resistance correction. so this should be 
```
-ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ) - (hd_sfc) * E0 + hd_sfc * (E0 - E) 
```
or
```
-ρ_sfc * Ch * windspeed(sc) * (cp_m * ΔT + ΔΦ) - (hd_sfc) * E
```
where `E` has been corrected for the surface resistance

Reply:
Ah - yes I get this now, I misinterpreted the notation `E` when compared to the atmos / sf src.

Reply:
This is resolved; `E0` is the outcome from SurfaceFluxes.jl therefore the above correction makes sense.